scripts/devops_tasks/create_coverage.py

```diff
@@ -53,7 +53,7 @@ def collect_tox_coverage_files():
 def generate_coverage_xml():
     if os.path.exists(coverage_dir):
         logging.info("Generating coverage XML")
-        commands = ["coverage", "xml", "-i", "--omit", '"*test*,*example*"']
+        commands = ["coverage", "xml", "-i"]
```

Comment:
Pls double check that it applies the exclusion correctly (automatically finding the coveragerc I mean).

Looks g2g other than that question!

Reply:
Verified with some pipeline runs here: https://github.com/Azure/azure-sdk-for-python/pull/29972#issuecomment-1515526565 🙂